### PR TITLE
feat(clearinghouse self-description): add admin UI to retrigger document creation

### PR DESCRIPTION
**Pull request by kunalgaurav-bmw**
## Description

Admin UI for managing SD document retriggering

## Why

The purpose of this ticket is to design and implement a Frontend (FE) User Interface that corresponds with the new backend features for managing SelfDescription (SD) documents. This interface will allow administrators to interact with the GET API endpoint and trigger the SD document creation job.

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/949

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: src/assets/locales/de/main.json [hunk 1]
```diff
@@ -1066,6 +1066,18 @@
       "metadata_url_required_error": "Metadaten-URL ist erforderlich",
       "metadata_url_invalid_error": "Bitte geben Sie eine gültige Metadaten-URL ein"
     },
+    "clearinghouseSelfDescription": {
+      "heading": "Status des Selbstbeschreibungsdokuments",
```
Comment (evegufy) [inline src/assets/locales/de/main.json:1070]: Hi @MaximilianHauer wouldn't be the proper term "Eigenerklärung"

File: src/components/pages/AdminClearingHouseSD/AdminClearingHouseSDElements.tsx [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+import {
+  type ComapnyDataType,
+  useFetchCompanyDataQuery,
+  useFetchConnectorsQuery,
+  useTriggerCompanyDataMutation,
+  useTriggerConnectorsMutation,
+  PAGE_SIZE,
+} from 'features/adminClearingHouseSD/adminClearingHouseSDApiSlice'
+import { error } from 'services/NotifyService'
+import { getRequireHttpsUrlPattern } from 'services/EnvironmentService'
```
Comment (evegufy) [inline src/components/pages/AdminClearingHouseSD/AdminClearingHouseSDElements.tsx:39]: as discussed, you still need to change to the proper value before merging

File: src/assets/locales/de/main.json [hunk 3]
```diff
@@ -1066,6 +1066,18 @@
...
+      "description": "Diese Seite bietet einen Überblick über den aktuellen Stand der Selbstbeschreibungen für Administratoren.",
+      "reprocess": "Wiederaufbereiten",
+      "complianceStatus": "Compliance-Status:",
+      "legalPerson": "Juristische Person",
+      "legalPersonDesc": "Übersicht über Unternehmen mit fehlender Selbstbeschreibung einer juristischen Person",
+      "connectors": "Anschlüsse",
```
Comment (evegufy) [inline src/assets/locales/de/main.json:1076]: as discussed with @MaximilianHauer , please change to "Connectors", in German as well

File: CHANGELOG.md [hunk 4]
```diff
@@ -28,6 +28,8 @@
   - implement new UI design for customer detail data overlay
 - **Company Subscription Management**
   - extended the subscription detail page for customer to include more app-related information [#1051](https://github.com/eclipse-tractusx/portal-frontend/pull/1051)
+- **Clearinghouse Self-Description**
+  - Admin UI for Managing SD Document Retriggering
```
Comment (evegufy) [inline CHANGELOG.md:32]: please make sure to add the comment in the changelog under the [`Unreleased` section](https://github.com/eclipse-tractusx/portal-frontend/blob/main/CHANGELOG.md#unreleased) and please also add the link to the pull request.

**Review by evegufy — CHANGES_REQUESTED** [review]

File: src/assets/locales/de/main.json (hunk 1 above)
Comment (kunalgaurav-bmw) [reply src/assets/locales/de/main.json]: Changes Done

File: src/assets/locales/de/main.json (hunk 3 above)
Comment (kunalgaurav-bmw) [reply src/assets/locales/de/main.json]: Changes Done

File: CHANGELOG.md (hunk 4 above)
Comment (kunalgaurav-bmw) [reply CHANGELOG.md]: Changes Done

File: src/assets/locales/de/main.json [hunk 5]
```diff
@@ -1066,6 +1066,18 @@
       "metadata_url_required_error": "Metadaten-URL ist erforderlich",
       "metadata_url_invalid_error": "Bitte geben Sie eine gültige Metadaten-URL ein"
     },
+    "clearinghouseSelfDescription": {
+      "heading": "Status der Eigenerklärung",
+      "description": "Diese Seite bietet einen Überblick über den aktuellen Stand der Selbstbeschreibungen für Administratoren.",
```
Comment (evegufy) [inline src/assets/locales/de/main.json:1071]:
```suggestion
      "description": "Diese Seite bietet einen Überblick über den aktuellen Stand der Eigenerklärungen für Administratoren.",
```

Please search the entire file for `Selbstbeschreibung` and correct with `Eigenerklärung`

**Review by evegufy — CHANGES_REQUESTED** [review]

**Review by MaximilianHauer — APPROVED** [review]

File: src/components/pages/AdminClearingHouseSD/AdminClearingHouseSDElements.tsx (hunk 2 above)
Comment (kunalgaurav-bmw) [reply src/components/pages/AdminClearingHouseSD/AdminClearingHouseSDElements.tsx]: Changes done

File: src/assets/locales/de/main.json (hunk 5 above)
Comment (kunalgaurav-bmw) [reply src/assets/locales/de/main.json]: Changes done

File: src/components/pages/AdminClearingHouseSD/AdminClearingHouseSDElements.tsx (hunk 2 above)
Comment (evegufy) [reply src/components/pages/AdminClearingHouseSD/AdminClearingHouseSDElements.tsx]: It's not fully enabled with your changes, @oyo wanted to look into it

**Review by oyo — APPROVED** [review]

**Review by oyo — CHANGES_REQUESTED** [review]
will update EnvironmentService to use injected env vars

**Review by oyo — APPROVED** [review]

**Review by evegufy — APPROVED** [review]

**Pull request merged**